slack-base/src/main/java/com/hubspot/slack/client/models/LiteMessageIF.java

```diff
@@ -15,22 +20,50 @@
 @HubSpotStyle
 @JsonNaming(SnakeCaseStrategy.class)
 public interface LiteMessageIF {
+
+  Logger LOG = LoggerFactory.getLogger(LiteMessage.class);
+
   String getType();
+
   Optional<String> getSubtype();
 
   Optional<String> getUser();
+
   Optional<String> getBotId();
+
   Optional<String> getUsername();
+
   String getText();
 
   List<Attachment> getAttachments();
+
   List<SlackFile> getFiles();
 
   @JsonProperty("ts")
   String getTimestamp();
+
   @JsonProperty("thread_ts")
   Optional<String> getThreadTimestamp();
 
   Optional<Integer> getReplyCount();
-  List<ReplySkeleton> getReplies();
+
+  Optional<List<String>> getReplyUsers();
+
+  Optional<Integer> getResplyUsersCount();
+
+  @JsonProperty("latest_reply")
+  Optional<String> getLatestReplyTimestamp();
+
+  /**
+   * @deprecated use {@link #getReplyUsers()} or {@link #getLatestReplyTimestamp()}
+   * These can be used to find the user ids and the last reply timestamp.
+   * Used to return a list of `ReplySkeleton`
+   */
+  @Deprecated
+  @Derived
```

Comment:
Do we want this to be `Derived`? If Slack is sending the value, we should return it and, if not, we can default to the empty list. This gives users a backwards compatible client

Reply:
If you're upgrading to this client, I like that it's always empty.
Rather than have something continue to work and just stop at some point in the future.

I'm also gonna remove `ReplySkeleton` as it probably has no reason to still be here.

Reply:
Actually looks like `LiteMessage` is used in other places, so probably safe to just do what you suggested. If they are sending it in other places, it will still be used.

Reply:
The goal of this update is to migrate off of `getReplies`. As written, if a user updates to this version, the `getReplies` method still exists in the model. If users try to use it, they either get what they except (before Oct 18th) or get back incorrect data (empty list). 

The fallback to the empty list is not accurate and can lead to unintended logic. This type of deprecation isn't a great experience. If we leave in this deprecated functionality, we should take the side of failing fast to prevent potentially costly misuses. 

I see 2 options

1) We remove the `getReplies` method entirely so users never accidentally call it. Anyone upgrading will have to migrate off `getReplies` since old code won't compile. This is explicit and removes any chance of errors. 

2) We expose a deprecated `getReplies` and log an ERROR whenever this method is called. If Slack no longer supplies this field and `getReplies` is called, we throw a RuntimeException. This will be harder to do with an immutable, but can be useful as users transition off `getReplies`.

I prefer option 1.

Reply:
So the issue is that `LiteMessage` is part of tons of other model objects. It looks like it's not just for conversations. Releasing this and having it log an error along with marking it as `@Deprecated` seemed like the safest thing to do here.

I'm gonna reach out to Slack and see if it's just the one specific place when https://api.slack.com/methods/conversations.replies is called where replies is being removed, or if it's other places. It may be that our `LiteMessage` needs to be forked for the `conversations.replies` case and all others.

Reply:
https://hubspot.slack.com/help/requests/2419401 They think this is ONLY `conversations.replies`, but waiting to have them confirm in case there are other places (beyond the `message_replied` event)

Reply:
Ok, now they say it's everything that returns this `replies` object, so back to `LiteMessage` where the change is.

Reply:
And made the default an empty array just to be safe.. So essentially back to what we had originally done.